internal/cli/get.go

```diff
@@ -80,152 +80,101 @@ func NewCmdGet() *cobra.Command {
 			if cmd.Flags().Lookup("continue").Changed {
 				cont = &o.Continue
 			}
-			return RunGet(kind, name, labelSelector, fleetName, o.Output, limit, cont)
+			return RunGet(kind, name, labelSelector, owner, o.Output, limit, cont)
 		},
 		SilenceUsage: true,
 	}
 
-	cmd.Flags().StringVarP(&o.FleetName, "fleetname", "f", o.FleetName, "fleet name selector for listing devices")
+	cmd.Flags().StringVar(&o.Owner, "owner", o.Owner, "filter by owner")
 	cmd.Flags().StringVarP(&o.LabelSelector, "labelselector", "l", o.LabelSelector, "label selector as a comma-separated list of key=value")
 	cmd.Flags().StringVarP(&o.Output, "output", "o", o.Output, "output format (yaml)")
 	cmd.Flags().Int32Var(&o.Limit, "limit", o.Limit, "the maximum number of results returned in the list response")
 	cmd.Flags().StringVar(&o.Continue, "continue", o.Continue, "query more results starting from the value of the 'continue' field in the previous response")
 	return cmd
 }
 
-func RunGet(kind, name string, labelSelector, fleetName *string, output string, limit *int32, cont *string) error {
+func RunGet(kind, name string, labelSelector, owner *string, output string, limit *int32, cont *string) error {
```

Comment:
context is a critical component of graceful termination anytime we are Background()ing context we should ask why,
```suggestion
func RunGet(ctx context.Context, kind, name string, labelSelector, owner *string, output string, limit *int32, cont *string) error {
```

Reply:
I probably have some golang knowledge gap here, but why should we pass ctx if we don't use it?

Reply:
All of the calls below plumb context.Background() I assume they used it.

Reply:
Ah I see its cli vs server but lets say you wanted to enforce a timeout. You would want a real context here.

Reply:
OK I see now.  I didn't see all the context.Background() below.  Will fix, thanks!

Reply:
Fixed in new commit